lib/src/drive-constraints.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2016 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const _ = require('lodash');
+
+module.exports = class DriveConstraints {
```

Comment:
This requires using `new` when using this class directly, which is not ideal. 

I think you can declare it the usual NodeJS way (using `exports`) and Angular does the right thing out of the box when creating a service out of it (let me know if its not the case and we'll sort it out somehow):

```
exports.isDriveLocked = (drive) => { ... }
exports.isSystemDrive = (drive) => { ... }
```